pcap-mode.el

```diff
@@ -362,6 +374,32 @@ Argument FILTER-VALUE corresponds to the exact set of filters passed to `pcap-mo
   (if pcap-mode-reload-pcap-when-filter-changes
       (pcap-mode-reload-file)))
 
+(defun pcap-mode-set-named-filter (filter-name)
+  "Choose a predefined filter and apply it.
+Filters are obtained from the `pcap-mode-dfilters-file'."
+  (interactive (list (ido-completing-read
```

Comment:
This requires ido mode, but there's no hint that such a change is required.  Please use completing-read.  You may also choose to write a function which detects the presence of ido-mode and prefers it vs. completing-read, with a fallback to completing-read.  Example below is completly untested:

(defun pcap-mode--completing-read ()
 "Returns the preferred completing read function."
  (if (fboundp ido-completing-read) ido-completing-read
    completing-read))